tests/lib/lore_test.sh

```diff
@@ -170,4 +195,102 @@ function test_extract_metadata_from_patch_title_rfc()
   assertEquals "($LINENO)" "$output" "$expected"
 }
 
+function test_create_lore_bookmarked_file()
+{
+  rm "${BOOKMARKED_SERIES_PATH}"
+
+  create_lore_bookmarked_file
+
+  assertTrue "($LINENO) - Local bookmark database was not created" "[[ -f ${BOOKMARKED_SERIES_PATH} ]]"
+}
+
+function test_add_series_to_bookmark()
+{
+  local target_patch1="someseries1"
+  local target_patch2="someseries2"
+  local target_patch3="someseries3"
+  local download_dir_path="somedir"
+  local output
+  local count
+
+  add_series_to_bookmark "${target_patch1}" "${download_dir_path}"
+  output=$(cat "${BOOKMARKED_SERIES_PATH}")
```

Comment:
You can use something more optimal than `cat` here; check https://kworkflow.org/content/codingstyle.html#command-substitution-and-arithmetic-expression .

Reply:
Nice!